generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -205,7 +255,7 @@ class FireSimCVA6Config extends Config(
 //* Multiclock Configurations
 //*********************************************************************************/
 class FireSimMulticlockRocketConfig extends Config(
-  new chipyard.config.WithTileFrequency(6400.0) ++ //lol
+  new chipyard.config.WithTileFrequency(6000.0) ++ //lol
```

Comment:
If the default is FireSimDefaultFrequency (2 GHz tile, 1 GHz everything else) , then do we still need this config around?

Reply:
@alonamid Yeah, you can delete these, and delete the one "multiclock" scalatest, since they are all multiclock now. You'll need to remove it from CI too.